browsers/webpack.config.js

```diff
@@ -93,6 +93,18 @@ function rollbarPlugins() {
   }
 }
 
+function getVersionName(isProduction) {
+  if (isProduction) {
+    return process.env.npm_package_version;
+  } else if (process.env.ENVIRONMENT === "staging") {
+    return `${process.env.npm_package_version}-alpha+${process.env.SOURCE_VERSION}`;
```

Comment:
String interpolation is more readable than string addition "+"

Reply:
I generally disagree with that, especially when the string is short compared to the variable names. Here you can see exactly the 3 pieces being merged without having to check whether the line starts with ``return `${process`` or ``return ` ${process``. 

```suggestion
    return process.env.npm_package_version + '-alpha+' + process.env.SOURCE_VERSION;
```

<code>`${</code>  is kind of noisy compared to… no extra characters at the beginning of the line.

I generally think `string ${interpolation} is only useful on ${longer} sentences`

Reply:
I can see you point in this case, but in general disagree (mostly because of longer sentences being the norm

ESLint has rule support for requiring templates https://eslint.org/docs/rules/prefer-template. I guess you could define a custom rule that counts number of characters in expressions relative to the literals, but probably not worth the effort 😄